controllers/domain/getDomain.js

```diff
@@ -37,53 +40,79 @@ module.exports = async (req, res) => {
         };
 
         const resp = await getDomain(query);
-        let feeds = resp.results
+        let feeds = resp.results;
 
         for (let i in feeds) {
           let item = feeds[i];
+          const newslink = await NewsLink.findOne({
+            where: { news_link_id: item.content.news_link_id },
+          });
+
+          if (newslink.priority === 2) {
+            offset++;
+            continue;
+          }
```

Comment:
mas @falhar this is too heavy, since for each requested item we call to DB
Getstream news link from rss has priority embedded, so you don't have to call DB.
<img width="475" alt="image" src="https://github.com/BetterSocial/user-api/assets/55779793/e31ef3f4-9d2f-4c7c-bcd9-0e823e78cb00">